pod/annotations.go

```diff
@@ -158,8 +158,15 @@ const (
 	// container annotations (specified on a pod about a container)
 	// Specific containers indicate they want to set something by appending
 	// a prefix key with their container name ($name.containers.netflix.com).
+	// Use the pod.ContainerAnnotation() function to help construct the annotation key
 	AnnotationKeySuffixContainers        = "containers.netflix.com"
 	AnnotationKeySuffixContainersSidecar = "platform-sidecar"
+	// AnnotationKeySuffixContainersSecurityCapabilities is a per-container annotation that sets
+	// the Titus SecurityCapabilities setting for that container.
+	// Values of this annotation should be the string value in the protobuf enum,
+	// but without the `ContainerCapabilities` string.
+	// A full example would be "security-capabilities.containers.netflix.com/mycoolsidecar: FUSE"
+	AnnotationKeySuffixContainersSecurityCapabilities = "security-capabilities"
```

Comment:
do you think I should put the word `security` in here at all?
I guess I didn't put it in the protobuf, so why should I have it here?

Reply:
yes, I'm +1 on leaving it out.